### PR TITLE
Kpssmerge1

### PR DESCRIPTION
**Pull request by shravanisamala**

### CODE REVIEW

**Review by neilhatfield — CHANGES_REQUESTED** [review]
@shravanisamala Please pull my recent commit (https://github.com/EducationShinyAppTeam/Counting_Techniques/commit/8551d3e3158e1f1f03d2bc68350e197c9829ba91) so that you can look through the code for "Fix:" and "Address:" labels on my comments. (Note: if something happened several times throughout the code, I did not mark each instance; you'll need to carry my comments throughout the code.)

File: app.R [hunk 1]
```diff
@@ -4,18 +4,31 @@ library(shinydashboard)
 library(shinyBS)
 library(shinyWidgets)
 library(boastUtils)
-library(shinyjs)
-library(data.table)
+# library(shinyjs) 
+## Fix: I don't believe that shinyjs is actively being used anywhere in the app
+# library(data.table)
+## Fix: The question banks you're loading are not large enough to require the use of
+## the data.table package and fread. Switch to read.csv
```
Comment (shravanisamala) [inline app.R:8]: @neilhatfield Hi Dr. Hatfield, So I tried changing this from fread to read.csv and whenever I do this, the first picture in the set of cards does not render and instead just says something like "7-spades.png". Should I keep it as fread? Or do you know why maybe this is occuring?

Comment (neilhatfield) [reply app.R:8]: @shravanisamala  I'm unable to replicate what you're describing. I used `read.csv(file = "www/pokerquestionbank.csv")`, `read.csv(file = "www/pokerquestionbank.csv", header = TRUE)`, and `read.csv(file = "pokerquestionbank")` (after moving the file to the main level). All of them resulted in full sets of cards rendering (including those with the question mark cards); that is, no broken images or just text listed.

Comment (shravanisamala) [reply app.R:8]: @neilhatfield Hi Dr. Hatfield, It is still happening to me so maybe we can take a look at it this Friday during the meeting. Thanks!

Comment (neilhatfield) [reply app.R:8]: Sounds like a plan

**Review by neilhatfield — APPROVED** [review]

**Pull request merged**